### PR TITLE
Updated delete sections.

### PR DESCRIPTION
**Pull request by dcoplowe**
# Description

Migrated delete information to OAS. 

## Type of change

Please delete options that are not relevant.

- [x] New document(s)/updating existing
- [ ] Fixes
- [ ] Styling
- [ ] Bug fix (non-breaking change which fixes an issue)

## Reviews and merging

You are responsible for getting your PR merged. Address review comments promptly and **make sure to merge the PR when ready**.
Feel free to 'Enable automerge' - your PR will automatically merge when accepted and passing the build.

### CODE REVIEW

File: docs/integrations/accounting/quickbooksonline/accounting-quickbooksonline-delete-journal-entries.md
Comment (d-laing) [inline docs/integrations/accounting/quickbooksonline/accounting-quickbooksonline-delete-journal-entries.md:1]: I feel there should be a link or reference to the functionality from here. Especially when delete endpoints are at such an early stage.

File: docs/integrations/accounting/netsuite/accounting-netsuite-how-deleting-bill-payments-works.md [hunk 1]
```diff
@@ -1,40 +1,9 @@
 ---
-title: "Delete Bills and Bill payments from NetSuite"
-sidebar_label: "Delete Bills and Bill payments"
-description: "Learn how to delete Bills and Bill payments from NetSuite using the deletion endpoints."
+title: "How deleting Bill payments works"
+sidebar_label: "How deleting Bill payments works"
```
Comment (d-laing) [inline docs/integrations/accounting/netsuite/accounting-netsuite-how-deleting-bill-payments-works.md:3]: "Delete Bill payments" is maybe more suitable for the sidebar.

**Review by d-laing — CHANGES_REQUESTED** [review]

**Comment by mcclowes** [comment]
Isn't the point of the changes you've made that we don't have a QB OR Netsuite specific page on deletion?

File: sidebars/integrations.js [hunk 2]
```diff
@@ -99,7 +99,7 @@ module.exports = [
           "integrations/accounting/netsuite/accounting-netsuite",
           "integrations/accounting/netsuite/accounting-netsuite-setup",
           "integrations/accounting/netsuite/company-linking-journey",
-          "integrations/accounting/netsuite/accounting-netsuite-delete-bills-billpayments",
+          "integrations/accounting/netsuite/accounting-netsuite-how-deleting-bill-payments-works",
```
Comment (mcclowes) [inline sidebars/integrations.js:102]: Shouldn't be under netsuite

**Review by mcclowes — CHANGES_REQUESTED** [review]

**Review by mcclowes — APPROVED** [review]

**Pull request merged**